server/bin/fxa-content-server.js

```diff
@@ -102,6 +103,18 @@ function makeApp() {
 
   app.use(require('express-able')(ableOptions));
 
+  // web fonts served from a CDN will not be rendered
+  // unless CORS headers are present since the fonts
+  // are considered to be used by a 3rd party domain.
+  var webFontCorsOptions = {
+    origin: config.get('public_url')
+  };
+
+  app.options(/\.woff$/, cors(webFontCorsOptions));
+  app.get(/\.woff$/, cors(webFontCorsOptions));
+  app.options(/\.ttf$/, cors(webFontCorsOptions));
+  app.get(/\.ttf$/, cors(webFontCorsOptions));
```

Comment:
Since you're already using regex's, not sure if you can reduce this by 1/2 by using something like:

```js
app.options(/\.(woff|ttf)$/, cors(webFontCorsOptions));
app.get(/\.(woff|ttf)$/, cors(webFontCorsOptions));
```

Or possibly even something like:
```js
app.route(/\.(woff|ttf)$/)
  .options(cors(webFontCorsOptions))
  .get(cors(webFontCorsOptions));
```